rel-eng/atomic-enterprise-node.service

```diff
@@ -0,0 +1,21 @@
+[Unit]
+Description=Atomic Enterprise Node
+After=docker.service
+After=atomic-enterprise-master.service
+After=atomic-enterprise-sdn-master.service
+After=atomic-enterprise-sdn-node.service
+Wants=docker.service
+Documentation=https://github.com/openshift/origin
+
+[Service]
+Type=notify
+EnvironmentFile=/etc/sysconfig/atomic-enterprise-node
+Environment=GOTRACEBACK=crash
+ExecStart=/usr/bin/atomic-enterprise start node --config=${CONFIG_FILE} $OPTIONS
```

Comment:
To my previous comment, I meant we should not set GOMAXPROCS unless you
want to make it a config option and be opinionated on it.

On Mon, Aug 10, 2015 at 4:20 PM, Timothy St. Clair <notifications@github.com
> wrote:

> In rel-eng/atomic-enterprise-node.service
> <https://github.com/openshift/origin/pull/3955#discussion_r36679190>:
>
> > @@ -0,0 +1,21 @@
> > +[Unit]
> > +Description=Atomic Enterprise Node
> > +After=docker.service
> > +After=atomic-enterprise-master.service
> > +After=atomic-enterprise-sdn-master.service
> > +After=atomic-enterprise-sdn-node.service
> > +Wants=docker.service
> > +Documentation=https://github.com/openshift/origin
> > +
> > +[Service]
> > +Type=notify
> > +EnvironmentFile=/etc/sysconfig/atomic-enterprise-node
> > +Environment=GOTRACEBACK=crash
> > +ExecStart=/usr/bin/atomic-enterprise start node --config=${CONFIG_FILE} $OPTIONS
>
> Same comment re: GOMAXPROCS.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/openshift/origin/pull/3955/files#r36679190>.
>



-- 
Clayton Coleman | Lead Engineer, OpenShift